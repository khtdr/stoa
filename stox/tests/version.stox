#!/usr/bin/env stox -v
