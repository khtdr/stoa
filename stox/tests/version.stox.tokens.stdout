stox-2022.08.15
